src/modules/dashboard/index.tsx

```diff
@@ -94,121 +110,180 @@ const Dashboard = () => {
     return (
       categoriesList?.map((cate) => ({
         label: cate.name,
-        value: cate.name,
+        value: cate.id,
       })) || []
     );
   }, [categoriesList]);
   const { data: departmentsList, isLoading: departmentsFetching } =
     useDepartments(1);
-
+  console.log(departmentsList, 'list');
   const deptOptions = useMemo(() => {
     return (
       departmentsList?.map((dept) => ({
```

Comment:
Instead of using useMemo for this you can directly perform this transition in query hook itself
You can make use of 'select' configuration for useQuery hook

Reply:
but that same hook also use in department listing table page